backend/de.metas.datev/src/main/sql/postgresql/ddl/views/RV_DATEV_Export_Fact_Acct_Invoice.sql

```diff
@@ -172,10 +172,11 @@ BEGIN
             TaxAmtSource = t.TaxAmtSource * (-1),
             dr_account   = t.cr_account,
             cr_account   = t.dr_account
-        WHERE t.docbasetype IN ('APC', 'ARC');
+        WHERE t.docbasetype IN ('APC', 'ARI');
```

Comment:
clarify this logic,
consider renaming the p_IsSwitchCreditMemo flag, also in Datev export table